go/docker/build.sh

```diff
@@ -0,0 +1,19 @@
+cat > ./Dockerfile << EOF
```

Comment:
Having both `go/docker/` and `docker/` seems confusing, move to the same direcotry or use `docker/build_docker.sh`.